### PR TITLE
Origin/translation. Some minor translation. Specify zoning for future references.

### PR DESCRIPTION
**Pull request by dangan114**

### CODE REVIEW

**Review by JOOOEG123 — APPROVED** [review]
Looks good

File: src/assets/i18n/cn.json [hunk 1]
```diff
@@ -23,76 +23,78 @@
...
-    "archive_filter_cat_four_title": "",
-    "archive_filter_cat_five_title": "",
-    "archive_filter_reset_button": ""
+    "archive_searchbar_placeholder": "未知",
+    "archive_top_title": "未知",
+    "archive_filter_title": "未知",
+    "archive_filter_cat_one_title": "未知",
+    "archive_filter_cat_one_checkbox_one": "未知",
+    "archive_filter_cat_one_checkbox_two": "未知",
+    "archive_filter_cat_two_title": "未知",
```
[the remark this answers is not in the document]
Comment (yzhang54) [reply src/assets/i18n/cn.json:81]: emm, the translation looks fine, but they meant "Unknown" for "未知." Are they supposed to be unknown?

File: src/assets/i18n/cn.json [hunk 2]
```diff
@@ -23,76 +23,78 @@
...
   },
   "gallery": {
-    "gallery_top_title": "",
-    "gallery_top_cat_one_button": "",
-    "gallery_top_cat_two_button": "",
-    "gallery_top_cat_three_button": "",
-    "gallery_top_cat_four_button": "",
-    "gallery_top_cat_five_button": "",
-    "gallery_image_button": ""
+    "gallery_top_title": "画廊",
```
Comment (yzhang54) [inline src/assets/i18n/cn.json:92]: Hi, I think these translations are pretty iffy. Mind showing me where these are used tmr?

File: src/assets/i18n/cn.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dangan114) [reply src/assets/i18n/cn.json:81]: They are placeholders for now. The translation would be done later on when the UX team finalizes their design.

File: src/app/pages/gallery/gallery.component.ts [hunk 3]
```diff
@@ -99,21 +97,41 @@ export class GalleryComponent implements OnInit {
...
+      this.galleries.length = 0
+
+      this.galleries.push(data['gallery_top_cat_one_button'])
+      this.galleries.push(data['gallery_top_cat_two_button'])
+      this.galleries.push(data['gallery_top_cat_three_button'])
+      this.galleries.push(data['gallery_top_cat_four_button'])
+      this.galleries.push(data['gallery_top_cat_five_button'])
+      this.title = data['gallery_top_title']
+      this.imageButton = data['gallery_image_button']
+    })
```
[the remark this answers is not in the document]
Comment (dangan114) [reply src/app/pages/gallery/gallery.component.ts:129]: It makes more sense to me this way. like when someone looks at the site, they will see this is the first button, second button and so forth. We can discuss a better way for this when we meet tomorrow.

File: src/assets/i18n/cn.json (hunk 2 above)
Comment (dangan114) [reply src/assets/i18n/cn.json:92]: Just got some translations of Google Translate. It is not finalized yet. I only wanted to set up the code base for the translation so we only need to update the jsons later on.

**Review by weilonp — APPROVED** [review]

**Pull request merged**